perses/rjmc/geometry.py

```diff
@@ -96,46 +97,49 @@ def __init__(self, metadata=None, use_sterics=False, verbose=True):
         self.verbose = verbose
         self.use_sterics = use_sterics
 
-    def propose(self, top_proposal, current_positions, beta):
+    def propose(self, top_proposal, current_sampler_state, beta):
         """
         Make a geometry proposal for the appropriate atoms.
 
         Arguments
         ----------
         top_proposal : TopologyProposal object
             Object containing the relevant results of a topology proposal
+        current_sampler_state : openmmtools.states.SamplerState
+            The current positions and box vectors of the proposal
```

Comment:
Why is `beta` a float? Either we should be using unitless energies (implied units of `kT`) where `beta = 1`, or we should be passing unit-bearing quantities.